CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Contributing to RKE #
+
+Thanks for taking the time to contribute to RKE!
+
+Of course, contributing code is more than welcome! To keep things simple, if you're fixing a small issue, you can simply submit a PR and we will pick it up. However, if you're planning to submit a bigger PR to implement a new feature or fix a relatively complex bug, please continue reading.
+
+### Opening PRs and organizing commits
+PRs should generally address only 1 issue at a time. If you need to fix two bugs, open two separate PRs. This will keep the scope of your pull requests smaller and allow them to be reviewed and merged more quickly.
+
+Each PR needs to be linked to a Github issue in [rancher project](https://github.com/rancher/rancher) which provides as much detail as possible on the commit at its motivation. If it is a bug fix, please describe how to reproduce it in the issue. Make the team or Project Manager aware of the Github issue so that they can set the correct milestone. Milestones are very important for release and QA tracking.
```

Comment:
Minor wording changes, but I agree with @a-blender 's suggestion to establish PR template.

```suggestion
Each PR needs to be linked to a Github issue in [rancher project](https://github.com/rancher/rancher) which provides as much detail as possible on the change and its motivation. If it is a bug fix, please describe how to reproduce it in the issue. Make the Team or Project Manager aware of the GitHub issue so that they can set the correct milestone. Milestones are very important for the release and QA tracking.
```

Reply:
PR template now included